### DISCUSSION TITLE
Select dependencies that should trigger a patch update

### DISCUSSION THREAD

**Discussion by DanySK** [renovatebot/renovate#12835] [Request Help] [answered]
Hi,
I see renovate automatically picks `fix` or `chore` on npm projects. I'd like to be able to select a set of dependencies that trigger a new patch (or even minor) release, binding some dependency name to `fix` or even `feat` type commits automatically.
Is there any control knob for that?

edit: even for non-npm projects, e.g., using gradle + docker

**Comment by viceice** [comment, marked as answer]
semanticCommitType

https://gitlab.com/renovate-bot/renovate-runner/-/blob/main/renovate.json

**Comment by DanySK** [comment]
Thanks for the pointer!

Do I understand correctly that this should trigger minor updates if `foo` gets updated, regardless of the manager?
```json
  "packageRules": [
    {
      "description": "Trigger a minor update if foo-* gets updated",
      "matchPackagePatterns": ["^foo.*"],
      "semanticCommitType": "feat",
    }
```

Thanks!

**Reply by viceice** [reply]
Yes

**Reply by DanySK** [reply]
lovely. I'm trying this immediately.
